tools/packaging/kata-deploy/local-build/check-runtimeclasses.sh

```diff
@@ -44,4 +50,30 @@ if ! diff $generated_file $original_file; then
     echo "CHECKER FAILED: files $(pwd)/$generated_file (GENERATED) and $(pwd)/$original_file differ"
     exit 1
 fi
+popd
+
+#
+# Checking the lists of SHIMS in the deployment files are up-to-dated.
+#
+for yaml in kata-deploy/base/kata-deploy.yaml \
+            kata-cleanup/base/kata-cleanup.yaml;do
+    # Get the current list of shims
+    shim_list="$(yq ".spec.template.spec.containers[0].env[2].value" $yaml)"
```

Comment:
Not sure whether I'm not overly cautious, but using `.spec.template.spec.containers[0].env[] | select(.name=="SHIMS").value` would be safer